src/routes/ContentDetails/ContentDetails.tsx

```diff
@@ -0,0 +1,50 @@
+import { MainLayout } from '@/components';
+import { findContentValueByLabel } from '@/utils/cms';
+import { Tag, tidyClasses as tc } from '@jod/design-system';
+import { useTranslation } from 'react-i18next';
+import { useLoaderData } from 'react-router';
+import { LoaderData } from './loader';
+
+const ContentDetails = () => {
+  const { data } = useLoaderData<LoaderData>();
+  const { i18n } = useTranslation();
+  const date = data.dateCreated
+    ? new Intl.DateTimeFormat([i18n.language], {
+        dateStyle: 'medium',
+      }).format(new Date(data.dateCreated))
+    : '';
+
+  const taxonomies = data.taxonomyCategoryBriefs?.map((taxonomy) => taxonomy.taxonomyCategoryName);
+  const image = findContentValueByLabel(data, 'Kuva')?.image;
+  const description = findContentValueByLabel(data, 'Kuvaus')?.data;
+  const richTextClasses = tc([
```

Comment:
Since the article description is done using a rich text editor and contains HTML code, I put these classes to make it look nicer. Otherwise the content would be completely unstyled and hard to read.